solution-base/mongodb/scripts-sharded/create-app-user-sharded.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
* these scripts are named "-sharded" : so we can put them in the same folder as the other scripts
* there is lot of redundancy here between the sharded and non-sharded scripts:
   - for create-app-user, the diff seems to only be the `mongosh` (vs `mongo`) command --> we should align (check version to use either mongo or mongosh? find from some other env variable? automatically 'detect' which one is available? ...)
    - for set-default-write-concern, it does not depend on sharding but on the version of mongo... and furtunately we don't use it in non-sharded mongo deployment

Reply:
=> Done here : https://github.com/scality/Zenko/pull/2028/commits/c54d619978cc1070b11672c258e0b88d38cbbf15#diff-f3e6344f5181df0001c53dac90d9291cb2ff6105589ab8a3d277e4f379f3cc5bR3
Keeping only one file and switching commands